website/learn/effective-elvish.md

```diff
@@ -223,7 +223,7 @@ lorem
 456
 ```
 
-Note that it is more concise to write `mygrep ... < in.txt`.
+**Note:** that it is more concise to write `mygrep ... < in.txt`.
```

Comment:
It seems like you have misunderstood what a notable "note" comment is and how it should be used. Also, even if appropriate here you should have done a s/that it is/It is/ substitution.

Reply:
Fixed in 41ddf5ace14faa33521e500bded307fffe26fa7c